lib/net/seeds/testnet.js

```diff
@@ -1,8 +1,6 @@
 'use strict';
 
 module.exports = [
-  'aoihqqagbhzz6wxg43itefqvmgda4uwtky362p22kbimcyg5fdp54@172.104.214.189',
```

Comment:
Also don't remove these ones, just move them to the bottom and put a comment around them saying they were the original testnet nodes

Reply:
Will do!